subworkflows/local/assembly_hybrid/main.nf

```diff
@@ -21,22 +20,24 @@ workflow HYBRID_ASSEMBLY {
             .map { _id, meta_long, long_reads, meta_short, short_reads ->
                 if (meta_long.lr_platform == "OXFORD_NANOPORE" || meta_long.lr_platform == "OXFORD_NANOPORE_HQ") {
                     [meta_short, short_reads, [], long_reads]
-                } else {
+                }
+                else {
```

Comment:
you don't have to obey the formatter! you are free!

Reply:
It means I have to turn it off which is like 5 clicks or something :cry: